src/lib/crypto/krb/prng_os.c

```diff
@@ -91,3 +91,11 @@ krb5_c_random_make_octets(krb5_context context, krb5_data *outdata)
     }
     return 0;
 }
+
+krb5_error_code KRB5_CALLCONV
+krb5_c_random_os_entropy(krb5_context context, int strong, int *success)
+{
+    /* The OS RNGs that accept input from userspace do not increase the
+     * entropy estimate based on this new entropy. */
```

Comment:
I understand that, but that's just a harmless accident of the design when the OS PRNG is treated both as the source of seed entropy and the source of PRNG data.

It's perfectly okay to short-circuit that, but the comment doesn't make any sense; we weren't trying to increase the operating system's entropy estimate by reading from /dev/urandom (or /dev/random in some cases) and writing it back.